conanfile.py

```diff
@@ -0,0 +1,256 @@
+    user = "silabs"
+
+    # Basic Conan metadata
+    description = "Matter extension for Simplicity SDK Suite"
+    license = "www.silabs.com/about-us/legal/master-software-license-agreement"
+    author = "Silicon Laboratories Inc."
+    homepage = ""  # E.g.,: https://stash.silabs.com/projects/<space>/repos/<project>/browse/README.md
+    url = ""  # E.g.,: https://stash.silabs.com/projects/<space>/repos/<project>/browse
+    topics = "silabs"  # You can add more topics
+
+    # Python module for .slc files parsing/expansion
+    python_requires = "silabs_package_assistant/1.2.6@silabs"
+
+    # Custom Silabs metadata
+    sl_metadata = {
+        "slack_channel": "",
+        "team": "",
+        "confluence_doc": "",
+        "jira_project": "",  # E.g.,: https://jira.silabs.com/projects/<space>/summary
+        "maintainers": [
+            {"name": "NA", "email": "NA@silabs.com"},
+        ],
+    }
+
+    # Other attributes
+    # revision_mode = "scm"
+
+    # Custom SLT metadata
+    # Reference: https://confluence.silabs.com/spaces/SS/pages/669417743/SLT+options+in+conanfile.py
+    """
+    Available options and their values:
+    packageType (string):
+        - sdk: SDK packages
+        - toolchain: Toolchain packages
+        - apack: APACK packages
+        - tools: Tool packages
+        - system: System packages (not for user-installable packages)
+        - software: General software packages (default)
+    subPackage (boolean):
+        - Only applicable for archive packages
+        - Should be set to False for Conan packages
+    releaseNotesUrl (string):
+        - URL to the release notes document
+    compatibleVersion (string):
+        - Version compatibility information
+    sdkLtsTag (string):
+        - Long Term Support (LTS) tag for SDK packages
+    """
+    # Dictionary to declare properties
+    options = {
+        "compatibleVersion": ["ANY"],
+        "subPackage": [True, False],
+        "releaseNotesUrl": ["ANY"],
+        "packageType": ["ANY"],
+        "sdkLtsTag": ["ANY"],
+    }
+
+    # Dictionary to define properties values.
+    # Alternative is to set values in def configure(self) of recipe
+    default_options = {
+        "compatibleVersion": "ANY",
+        "subPackage": False,
+        "releaseNotesUrl": "",
+        "packageType": "sdk",
+        "sdkLtsTag": "",
+    }
+
+    def requirements(self):
+        pass
+
+    def layout(self):
+        pass
+
+    def deploy(self):
+        pass
+
+    def export(self):
+        pass
+
+    def package_id(self):
+        # Completely clear all the info, resulting ``package_id`` will be the same
+        self.info.clear()
+
+    def package(self):
+        # Define the source folder for the matter component
+        matter_folder = self.source_folder
+
+        # Define the files to be included in the package
+        files_to_package = {"License.txt"}
+        extra_files = []
+
+        slce_extra_file_path = "matter.slce.extra"
```

Comment:
This could be come a constant at the top of the file since we define it multiple times.